.rubocop.yml

```diff
@@ -24,6 +24,9 @@ Layout/MultilineMethodCallIndentation:
 Layout/EmptyLinesAroundAccessModifier:
   EnforcedStyle: only_before
 
+Layout/TrailingWhitespace:
+  AllowInHeredoc: true
```

Comment:
I don't massively care, but is this necessary? Or could the trailing whitespace be removed?

Reply:
I've added two suggests to make this specific to just this HEREDOC file. I'm agnostic to the change, but I'd want to also change it in the other rubocop places (I like consistent files) and I'm not sure we necessarily want to do that now (although, I don't care if it's troublefree 🙂)

Reply:
Having done that, can this be removed?

Reply:
d'oh!